pyClient/zeth/utils.py

```diff
@@ -74,6 +74,14 @@ def encode_abi(type_names: List[str], data: List[bytes]) -> bytes:
     return eth_abi.encode_abi(type_names, data)  # type: ignore
 
 
+def encode_eth_address(eth_addr: str) -> bytes:
+    """
+    Binary encoding of ethereum address to 32 bytes
+    """
+    # Strip the leading '0x' and hex-decode.
+    return extend_32bytes(bytes.fromhex(eth_addr[2:]))
```

Comment:
I don't see this function defined anywhere, there is however `hex_extend_32bytes`

Reply:
Thanks.  This was a mistake while rebasing.  Fixed.